Hint after pricing change (#1081)
uyuni-project/uyuni-docs: modules/installation/pages/general-requirements.adoc

//REMARK What about Uyuni?
ifeval::[{suma-content} == true]
You only require a valid registration code for {productname}, for example from a "SUSE Manager Lifecycle Management+" subscription.

> (For details, please see the our Terms and Conditions at https://www.suse.com/products/terms_and_conditions.pdf)

Reviewer: ```suggestion
For more information, see SUSE Terms and Conditions at https://www.suse.com/products/terms_and_conditions.pdf.
```
